infra/gcp/bash/ensure-terraform.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script ensures terraform service accounts for prow jobs and GCB are created.
+# DOCS: https://cloud.google.com/build/docs/securing-builds/configure-user-specified-service-accounts#cross-project_set_up
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
+    echo "usage: $0" > /dev/stderr
+    echo > /dev/stderr
+}
+
+if [ $# != 0 ]; then
+    usage
+    exit 1
+fi
+
+# The GCP project name.
+PROJECT=$(k8s_infra_project "prow" "k8s-infra-terraform")
+
+color 6 "Ensuring project exists: ${PROJECT}"
+ensure_project "${PROJECT}"
+
+PROJECT_SERVICES=(
+    iam.googleapis.com
+)
+
+# Enable GCB APIs
+color 6 "Ensure required services are enabled for: ${PROJECT}"
+
+ensure_only_services "${PROJECT}" "${PROJECT_SERVICES[@]}"
+
+# Create various service accounts for Terraform
+color 6 "Creating oci-proxy dev Service Account"
+ensure_service_account \
+    "${PROJECT}" \
+    "oci-proxy-dev" \
+    "oci proxy dev Terraform Account"
+
+color 6 "Creating oci-proxy prod Service Account"
+ensure_service_account \
+    "${PROJECT}" \
+    "oci-proxy-prod" \
+    "oci proxy prod Terraform Account"
+
+oci_proxy_dev_account="$(svc_acct_email "${project}" "${STAGING_SIGNER_SVCACCT}")"
```

Comment:
what's the purpose of `STAGING_SIGNER_SVCACCT` ?

Reply:
that is a typo should be oci-proxy-dev and oci-proxy-prod